network-api/networkapi/wagtailpages/models.py

```diff
@@ -623,6 +886,7 @@ class Homepage(MetadataPageMixin, Page):
         'Styleguide',
         'NewsPage',
         'ParticipatePage',
```

Comment:
do we need to remove `ParticipatePage`?

Reply:
We will eventually. Right now there will be two ParticipatePage templates so that we don't have to take down the current `/participate` page while we build the new one on prod.

Reply:
kk makes sense